src/constants/config.json

```diff
@@ -1,8 +1,62 @@
 {
-  "cartoDB": {
-    "mapVisualisationUrl": "https://shaunanoordin-zooniverse.cartodb.com/api/v2/viz/e04c2e20-a8a9-11e5-8d6b-0e674067d321/viz.json",
-    "dataTable": "wildcam_gorongosa_cameras_201601",
-    "dataLayerIndex": 1
+  "mapExplorer": {
```

Comment:
I reckon we should break this file out into separate json file, it's getting pretty big